cmd/juju/application/deploy.go

```diff
@@ -437,20 +438,19 @@ latter prefixed with "^", similar to the 'tags' constraint).
 
 When deploying bundles, machines specified in the bundle are added to the
 model as new machines. In order to use the existing machines in the model
-rather than create  new machines, the --use-existing-machines flag can be
-used. To specify a particular machine, the --bundle-machine flag can be used
-to specify a specific machine for  a particular machine in the bundle. These
-flags can be combined with the --bundle-machine flag taking precidence. For
-example, if there was a bundle that specified machines 1, 2, and 3, and the
-model had machines 1, 2, 3 and 4, the following deployment of the bundle would
-use machines 1 and 2 in the model for machines 1 and 2 in the bundle and use
-machine 4 in the model for the bundle machine 3.
+rather than create  new machines, the option --machine-map=existing can be
```

Comment:
double space in `create  new`? Odd thing to call out but may as well.

Reply:
fixed